tests/cmd/main_test.py

```diff
@@ -6,6 +6,7 @@
 import os.path
 
 import mock
+from six import StringIO
```

Comment:
I'd rather not add a new dependency just to get `StringIO`. Can you do
```
try:
    from io import StringIO
except ImportError:
    from StringIO import StringIO
```
instead?